mount/temp_unix.go

```diff
@@ -53,6 +56,9 @@ func CleanupTempMounts(flags int) error {
 	sort.Sort(sort.Reverse(sort.StringSlice(toUnmount)))
 	for _, path := range toUnmount {
 		if err := UnmountAll(path, flags); err != nil {
+			if errors.Cause(err) == unix.EBUSY {
+				continue
```

Comment:
Log the error? It may be lost if the last unmount succeeds

Maybe move the log here instead ? I was going to suggest to return for other kinds of error, but I can only think of EPERM, ENOENT & EINVAL as possible other errors which we may want to ignore on boot too

Reply:
i'm debating this, I don't really want to put logging in the mount pkg but...

Reply:
We can either continue on any error, aggregate them and return; Or return for any error immediately. :) It doesn't seem good to silently skip `unix.EBUSY`.

Reply:
Yes, we can just keep on wrapping it otherwise :)